pages/apis/rest_api/packages/packages.md

```diff
@@ -2,6 +2,49 @@
 
 The packages tokens API lets you create and manage packages in a registry.
```

Comment:
Hi @ensvo ,

Sorry I didn't pick this up earlier. There were too many other related PRs to manage getting merged into `main` correctly that I missed checking the more important aspect of these PRs—their content...

Anyhow, it looks like the word 'tokens' should be omitted from this line, since I'm not aware of anything called a 'package token'?

```suggestion
The packages API endpoint lets you create and manage packages in a registry.
```